apps/dapp-console-api/src/testhelpers/deployFaucetContracts.ts

```diff
@@ -0,0 +1,106 @@
+import { faucetAbi, faucetByteCode } from '../abi/Faucet'
+import {
+  adminFaucetAuthModuleAbi,
+  adminFaucetAuthModuleByteCode,
+} from '../abi/AdminFaucetAuthModule'
+import type { PrivateKeyAccount, PublicClient, WalletClient } from 'viem'
+
+export const deployFaucetContract = async (
+  publicClient: PublicClient,
+  walletClient: WalletClient,
+  account: PrivateKeyAccount,
+) => {
+  const faucetDeploymentHash = await walletClient.deployContract({
+    abi: faucetAbi,
+    bytecode: faucetByteCode,
+    account,
+    chain: walletClient.chain,
+    args: [account.address],
+  })
+
+  const receipt = await publicClient.waitForTransactionReceipt({
+    hash: faucetDeploymentHash,
+  })
+
+  return receipt.contractAddress!
+}
+
+export const deployOptimistFamContract = async (
```

Comment:
optimist fam was only a thing for the v0 version of the faucet, this can be deleted